pkg/controller/changetierrequest/changetierrequest_controller_test.go

```diff
@@ -273,17 +273,13 @@ func NewNSTemplateTier(tierName, revision, clusterResourcesRevision string, nsTy
 	namespaces := make([]v1alpha1.NSTemplateTierNamespace, len(nsTypes))
 	for i, nsType := range nsTypes {
 		namespaces[i] = v1alpha1.NSTemplateTierNamespace{
-			Type:        nsType,
-			Revision:    revision,
-			Template:    templatev1.Template{},
 			TemplateRef: nstemplatetiers.NewTierTemplateName(tierName, nsType, revision),
 		}
 	}
 	var clusterResources *v1alpha1.NSTemplateTierClusterResources
 	if clusterResourcesRevision != "" {
 		clusterResources = &v1alpha1.NSTemplateTierClusterResources{
-			Revision:    clusterResourcesRevision,
-			TemplateRef: nstemplatetiers.NewTierTemplateName(tierName, "clusterresources", clusterResourcesRevision),
+			TemplateRef: nstemplatetiers.NewTierTemplateName(tierName, toolchainv1alpha1.ClusterResourcesTemplateType, clusterResourcesRevision),
```

Comment:
I would rather use the string for the tests, but it's just a detail :-) 
```suggestion
			TemplateRef: nstemplatetiers.NewTierTemplateName(tierName, "clusterresources", clusterResourcesRevision),
```

Reply:
ah right, good point!